pkg/install/role.yaml

```diff
@@ -131,4 +135,4 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     namespace: acorn-image-system
-    name: acorn-image-system
```

Comment:
nit: Was this intentional?

Reply:
I believe the YAML file was auto formatted. I don't think it hurts anything.